docusaurus/docs/dev-docs/plugins/content-source-map.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Content Source Map plugin
+description: Use the Content Source Map plugin in combination with Vercel Visual Editing to make content edition more intuitive.
+sidebar_label: Content Source Map
+displayed_sidebar: devDocsSidebar
+---
+
+# Content Source Map plugin <EnterpriseBadge />
+
+The Content Source Map plugin adds content source maps support to a Strapi project.
+
+Content source maps are invisible values that include a link back to the field that generated the content.  When combined with [Vercel's Visual Editing](https://vercel.com/docs/workflow-collaboration/visual-editing) feature, content source maps allow navigating directly from the front-end website of a Strapi-based project to the source of a field in Strapi's admin panel, making editing content more intuitive.
+
+:::info
```

Comment:
I will add a prerequisites callout. Thanks!
However, since the plugin itself can't do anything without Vercel Visual's Editing, I think the description should be intricately linked to it, so I'll probably keep this in the introduction, too. What do you think? 😊

Reply:
Agreed! I think it looks great now! 👏